recipe-client-addon/test/browser/browser_ActionSandboxManager.js

```diff
@@ -79,28 +79,23 @@ add_task(async function testError() {
       await manager.runAsyncCallback("testCallback");
       ok(false, "runAsnycCallbackFromScript throws errors when raised by the sandbox");
     } catch (err) {
-      equal(err.message, "WHY", "runAsnycCallbackFromScript clones error messages");
+      is(err.message, "WHY", "runAsnycCallbackFromScript clones error messages");
     }
   });
 });
 
 add_task(async function testDriver() {
   const script = `
     registerAsyncCallback("testCallback", async function(normandy) {
-      return normandy;
+      return Object.keys(normandy);
     });
   `;
 
   await withManager(script, async manager => {
-    const sandboxDriver = await manager.runAsyncCallback("testCallback");
+    const sandboxDriverKeys = await manager.runAsyncCallback("testCallback");
     const referenceDriver = new NormandyDriver(manager);
-    equal(
-      sandboxDriver.constructor.name,
-      "NormandyDriver",
-      "runAsyncCallback passes a driver as the first parameter",
-    );
-    for (const prop in referenceDriver) {
-      ok(prop in sandboxDriver, "runAsyncCallback passes a driver as the first parameter");
+    for (const prop of Object.keys(referenceDriver)) {
+      ok(sandboxDriverKeys.includes(prop), `runAsyncCallback's driver has the "${prop}" property.`);
```

Comment:
Ah, that makes sense. Adding a comment here as to why this is comparing keys rather than just checking the objects might be a good idea.